src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -631,7 +631,7 @@ namespace
             max_samples->setToolTip(m_params_metadata.get_path("adaptive_tile_renderer.max_samples.help"));
             sublayout->addRow("Max Samples:", max_samples);
 
-            QDoubleSpinBox* noise_threshold = create_double_input("adaptive_tile_sampler.noise_threshold", 0.0001, 10000.0, 4, 0.02);
+            QDoubleSpinBox* noise_threshold = create_double_input("adaptive_tile_sampler.noise_threshold", 0.0001, 1000000.0, 4, 0.02);
```

Comment:
Not sure about this change, there was some kind of symmetry before: from 1/10000 to 10000.

Reply:
We makes sure that both spinBox are the same size with this

Reply:
That seems like a hack. I prefer that we set the ranges correctly than the spinbox have the same widths.